chain/consensus/filcns/upgrades.go

```diff
@@ -2609,43 +2609,56 @@ func PreUpgradeActorsV15(ctx context.Context, sm *stmgr.StateManager, cache stmg
 		workerCount /= 2
```

Comment:
I think I understand why we are doing a pre upgrade but lets make sure we're thinking the same thing.

The only reason is to speed up the base migration since the power actor state is negligible.

The premigration speeds up the migration by writing all 3M actors with new code cids to badger and only a small fraction of those will change.  So our badger writes will speed up.  But we don't actually cache base migration changes in memory so we're not gaining anything from caching.

@rvagg does this line up with your thinking?

Reply:
Yes. I decided that it would be best to document this in the code, so I added a comment at the top of this function for future readers.